todo_app/data/trello_items.py

```diff
@@ -0,0 +1,59 @@
+import requests
+import os
+
+
+class Item:
+    id = ''
+    status = ''
+    name = ''
```

Comment:
You don't need to include these here, they're all set by the constructor anyway.